src/terraform/modules/aws/custom/exocom/exocom-service/exocom-service.tf

```diff
@@ -2,7 +2,6 @@ module "task_definition" {
   source = "./exocom-task-definition"
 
   command               = "${var.command}"
-  container_port        = "${var.container_port}"
```

Comment:
Let's keep this variable in, that way we are in line with behavior on Exocom: container ports come online at port 80 by default but can be overridden if one is passed down. Removing this line will prevent us from passing one down at all. Just removing `container_port = "3100"` in `exocom.tf` and in the corresponding test is sufficient.